roles/openshift_hosted/README.md

```diff
@@ -33,13 +33,17 @@ variables also control configuration behavior:
 
 | Name                                         | Default value | Description                                                                  |
 |----------------------------------------------|---------------|------------------------------------------------------------------------------|
-| openshift_hosted_registry_glusterfs_swap     | False         | Whether to swap an existing registry's storage volume for a GlusterFS volume |
-| openshift_hosted_registry_glusterfs_swapcopy | True          | If swapping, also copy the current contents of the registry volume           |
```

Comment:
I completely disagree with this removal. If we are documenting the ips and path values here, all `reigistry_strorage_glusterfs` variables should be documented here. Copy all values from `roles/openshift_storage_glusterfs/README.md` into here and change said file to reference this one with something like the following:

`Additionally, this role's behavior responds to several registry-specific variables in the [openshift_hosted role](RELATIVE_URL).`